### ISSUE TITLE
Randomly occurring (racecondtiony) bug in log4perl tests

### ISSUE THREAD

**Issue by kentfredric** [plack/Plack#555]
Unable to reproduce with either test independently: 
```
prove -j10 -lvr t/Plack-Middleware/log4*
===(       1;0  1/?  0/? )==============================================ok
t/Plack-Middleware/log4perl.t ........... 
not ok 1
ok 2

#   Failed test at t/Plack-Middleware/log4perl.t line 38.
#                   ''
#     doesn't match '(?^:INFO - This is info)'
# Looks like you failed 1 test of 2.
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests 

Test Summary Report
-------------------
t/Plack-Middleware/log4perl.t         (Wstat: 256 Tests: 2 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
Files=2, Tests=4,  0 wallclock secs ( 0.03 usr  0.00 sys +  0.20 cusr  0.04 csys =  0.27 CPU)
Result: FAIL
```

**Comment by kentfredric** [comment]
A quick guess suggests that they're both using this file and its a problem:

```
my $test_file = "t/Plack-Middleware/log4perl.log";
```

**Comment by miyagawa** [comment]
This has been fixed in #545
